src/main/java/org/example/springproject/exception/CustomGlobalExceptionHandler.java

```diff
@@ -48,4 +45,13 @@ public ResponseEntity<Object> handleEntityNotFoundExceptions(EntityNotFoundExcep
         body.put("error", ex.getMessage());
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
+
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(EntityNotFoundException ex) {
```

Comment:
pay attention to the exception specified in the annotation params and within the method params